tools/GenerateRollUpModule.ps1

```diff
@@ -69,17 +69,24 @@ elseif ($VersionState.Equals([VersionState]::Valid) -or $VersionState.Equals([Ve
     if (-not (Test-Path $GraphModuleLocation)) {
         New-Item -Path $GraphModuleLocation -Type Directory
     }
+    $AllowPreRelease = $false
+    if($ModulePreviewNumber -ge 0) {
+        $AllowPreRelease = $true
+    }
 
     # Add auth module as a dependency.
-    $ExistingAuthModule = Find-Module "Microsoft.Graph.Authentication"
-    Install-Module $ExistingAuthModule.Name -Repository $RepositoryName -AllowPrerelease -Force
+    Find-Module "Microsoft.Graph.Authentication" -Repository $RepositoryName -AllowPreRelease:$AllowPreRelease
```

Comment:
Duplicate of the line below.

Reply:
I added the logging statements to see what the pipeline was doing. I will remove them when successful